GRD-85034

apurva-birajdar opened the pull request:
Publish all internal changes to public. 
Verified by QA .

PratikshaSonawane commented on filter-plugin/logstash-filter-aurora-mysql-guardium/auroraMysqlCloudwatch.conf:
```diff
@@ -17,31 +17,36 @@ input{
 		event_filter => ""
 		type => "AuroraMysql"
 		add_field => {"account_id" => "<account_id>"}
+        #Unmask is an optional parameter. To get unmasked logs, you need to set unmask = true.
```
Please remove these lines this fix is yet to be tested

apurva-birajdar replied:
done.

ofer-haim commented:
Shouldn't we upgrade the plugin version?

apurva-birajdar commented:
> Shouldn't we upgrade the plugin version?

Hi Ofer, We have not updated zip its just few checks that we added and removed the incorrect ones.
